EthOn.rdf

```diff
@@ -1954,7 +1954,7 @@
         <rdfs:subClassOf rdf:resource="http://xmlns.com/foaf/0.1/Agent"/>
         <simpleDefinition>A person or other entity controlling one or more external accounts, which can interact with Ethereum by signing transactions with a public-private keypair.</simpleDefinition>
         <suggestedStringRepresentation xml:lang="en">ExternalActor</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">A person or other entity able to interface with an Ethereum node. It can interact with Ethereum only by depositing signed transactions and inspecting the blockchain and associated state. An external actor controls the private-public keypair of one or more external accounts. While it is assumed that the ultimate external actor will be human in nature, software tools will be used in its construction and dissemination.</rdfs:comment>
+        <rdfs:comment xml:lang="en">A person or other entity able to interface with an Ethereum node. An external actor controls the private-public keypair of one or more external accounts. It can change the state of the blockchain by sending signed and valid transactions. Without using a keypair, external actors can also inspect and read from the blockchain and its state.</rdfs:comment>
```

Comment:
This flows better as a paragraph, improves clarity without removing any information